src/js/helpers/Util.js

```diff
@@ -5,6 +5,22 @@ var consecutiveNumber = 0;
 // method can't be *call*ed on different objects in IE.
 var toString = Object.prototype.toString;
 
+function assignWithAccessors(target, ...sources) {
```

Comment:
Good idea to move the functionality to a separate function. This is good if we ever want to change the behaviour of extendObject. But still keep assignWithAccessors.